robyn/router.py

```diff
@@ -79,25 +89,42 @@ def add_route(
         handler: Callable,
         is_const: bool,
         exception_handler: Optional[Callable],
+        default_response_headers: List[Header],
     ) -> Union[Callable, CoroutineType]:
+        response_headers = {
+            d[0]: d[1] for d in (h.as_list() for h in default_response_headers)
+        }
```

Comment:
Hey @IdoKendo 👋 

Great work as usual 😄  However, can we do something else instead of nested interpolation?

Reply:
Hey @sansyrox 👋 
Haha, yeah I was also not very satisfied with this line, but honestly the `Header` API is kinda awkward, so it's a bit tricky to do it any other way without introducing tons of boilerplate. I can try to fiddle with it so it's a bit nicer, or we can open a separate issue for it. WDYT?

Reply:
@IdoKendo , let us just use two for loops for now. 

> I can try to fiddle with it so it's a bit nicer, or we can open a separate issue for it. WDYT?

I think we should definitely investigate 😄 But I will not consider it a blocker for this PR